<launch>
  <node pkg="sr_visual_servoing" name="analyse_moving_object" type="analyse_moving_object">
    <rosparam command="load" file="$(find sr_visual_servoing)/param/prediction_model.yaml"/>
  </node>

  <node pkg="sr_visual_servoing" name="visual_servoing" type="visual_servoing"/>
</launch>